app/elements/common/register/register-page.html

```diff
@@ -74,6 +100,26 @@
 				</paper-input>
 				<paper-input type="password" label="Repeat password" value="{{repeatedPassword}}" autofocus="[[!externalWidget]]" on-keydown="onKeydown" required></paper-input>
 			</form>
+			<div id="socialSignInWrapper">
```

Comment:
Can't we make this thing a component? It looks quite similar to what we have for login...